data/names/ru.json

```diff
@@ -3031,22 +3033,27 @@
       "Айверсон",
       "Айви",
       "Айерс",
-      "Айзекс",
       "Айзек",
+      "Айзекс",
       "Айкен",
       "Айрлэнд",
       "Акерман",
+      "Акерс",
       "Акино",
       "Акинс",
       "Акоста",
       "Акуна",
+      "Аланиз",
       "Аланис",
       "Аларкон",
       "Алвес",
       "Александер",
+      "Александр",
       "Алеман",
       "Али",
       "Аллен",
+      "Аллея",
```

Comment:
I notice that a lot of the names on the list are direct translations of common nouns: in this case, "alley" (as in a narrow path between two buildings). Common nouns can also be names (such as Bravery or Virtue), in which case they would be transcribed rather than translated.

Here, presuming I'm right and the original was indeed the name Alley, it would be spelled phonetically. "Алли", I would think? I'm not positive whether the first letter would be "А" or "Э", since Russian has a tendency to transcribe the `[æ]` sound as "э": cat (as in the animal) = "кэт".